src/OpenSage.Game/Content/SubsystemLoader.cs

```diff
@@ -108,6 +108,9 @@ public void Load(Subsystem subsystem)
                 case Subsystem.InGameUI:
                     LoadFiles(@"Data\INI\InGameUI.ini");
                     break;
+                case Subsystem.AttributeModifiers:
```

Comment:
I think those subsystems should not be loaded in the first place for Generals. See ContentManager.cs

Reply:
done